crates/ruff/resources/test/fixtures/pylint/redefined_loop_name.py

```diff
@@ -0,0 +1,96 @@
+# For -> for, variable reused
```

Comment:
(Just out of interest, was this taken from Pylint or is it original?)

Reply:
It's original.